modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Iroha2Client.kt

```diff
@@ -79,14 +82,36 @@ open class Iroha2Client(
             .status.value
     }
 
+    /**
+     * Sends configuration request
+     */
+    suspend inline fun <reified T> configuration(
+        fieldName: ConfigurationFieldType = ConfigurationFieldType.VALUE,
+        fieldValue: Collection<String>? = null
+    ): T {
+        val response: HttpResponse = client.value.get("$peerUrl$CONFIGURATION_ENDPOINT") {
+            contentType(ContentType.Application.Json)
+            body = mapOf(fieldName.fieldName to fieldValue)
+        }
+        return response.receive()
+    }
+
     /**
      * Sends status check request
      */
     suspend fun status(): Map<*, *> {
         return client.value
             .get<HttpResponse>("$telemetryUrl$STATUS_ENDPOINT")
-            .content.readUTF8Line()
-            .let { mapper.readValue(it, Map::class.java) }
+            .receive()
+    }
+
+    /**
+     * Sends metrics request
+     */
+    suspend fun metrics(): String {
```

Comment:
That is a point to making metrics just `String`? For whom it will be helpful? 
I think better to create separate gradle module with enhancement of `Iroha2Client` (for example `AdminIroha2Client`) and move there all "admin" endpoints (healthcheck, status, configuration). And implement our own simple metrics parser. It is not difficult due Prometheus format is pretty simple